tests/test_transplants.py

```diff
@@ -0,0 +1,113 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from urllib.parse import parse_qs
+
+import json
+import requests_mock
+
+from unittest import mock
+
+from landoapi import app as _app
+from landoapi.models.storage import db
+from landoapi.models.transplant import (Transplant, TRANSPLANT_JOB_STARTED)
+from tests.utils import *
+from tests.canned_responses.phabricator.revisions import *
+from tests.canned_responses.phabricator.users import *
+from tests.canned_responses.phabricator.repos import *
+from tests.canned_responses.lando_api.revisions import *
+
+
+@pytest.mark.usefixtures('client_class')
+class TestTransplants:
+    def setup_method(self):
+        conn = _app.create_app('/version.json')
+        with conn.app.app_context():
```

Comment:
I actually think this is ok, or at least not something we have to worry about for now. Hopefully we find a solution soon :)